### PR TITLE
PLAT-477: text format support

### PR DESCRIPTION
**Pull request by sexyhamster**
**What I did**

- added support for text/json go test output
- fixed Parallel test output for wrong test

I didn't touch all the rest, due that we don't have integration tests here

### CODE REVIEW

File: source/utils.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+)
+
+func TryExtractTest(line []byte) (string, bool) {
+	if bytes.HasPrefix(line, []byte("    Test")) && len(line) > 8 && bytes.Index(line[8:], []byte(":")) != -1 {
+		part := line[4:bytes.Index(line, []byte(":"))]
```
Comment (ElSacramento) [inline source/utils.go:16]: magic numbers - 4, 8

File: source/text/parser_test.go [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/insolar/testrail-cli/source"
+)
+
+func TestParser_Parse(t *testing.T) {
+	return
```
Comment (skudasov) [inline source/text/parser_test.go:19]: the test is failing, why return?

File: source/json/example_log_test.go [hunk 3]
```diff
@@ -0,0 +1,90 @@
+//  Copyright 2020 Insolar Network Ltd.
+//  All rights reserved.
+//  This material is licensed under the Insolar License version 1.0,
+//  available at https://github.com/insolar/testrail-cli/LICENSE.md.
+
+package json
+
+import "github.com/insolar/testrail-cli/source"
+
+var expectedLog = []source.TestEvent{
```
Comment (skudasov) [inline source/json/example_log_test.go:10]: move to a file as in text module?

File: source/text/parser.go [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+
+		return res
+	}
+
+	res := make([]source.TestEvent, 0)
+
+	// Special case for entirely skipped test binary: "?   \tpkgname\t[no test files]\n" is only line.
+	// Report it as plain output but remember to say skip in the final summary.
+	if bytes.HasPrefix(line, skipLinePrefix) && bytes.HasSuffix(line, skipLineSuffix) && len(c.report) == 0 {
+		c.pkg = string(line[5 : bytes.Index(line[5:], []byte{0x09})+1])
```
Comment (ElSacramento) [inline source/text/parser.go:176]: magic number 5

File: source/text/parser.go [hunk 5]
```diff
@@ -0,0 +1,309 @@
...
+
+	// Parse out action and test name.
+	i := 0
+	if actionColon {
+		i = bytes.IndexByte(line, ':') + 1
+	}
+	if i == 0 {
+		i = len(updates[0])
+	}
+	action := strings.ToLower(strings.TrimSuffix(strings.TrimSpace(string(line[4:i])), ":"))
```
Comment (ElSacramento) [inline source/text/parser.go:241]: magic number 4

File: source/utils.go (hunk 1 above)
Comment (sexyhamster) [reply source/utils.go]: fixed

File: source/json/example_log_test.go (hunk 3 above)
Comment (sexyhamster) [reply source/json/example_log_test.go:10]: moved

File: source/text/parser.go (hunk 4 above)
Comment (sexyhamster) [reply source/text/parser.go]: added comment, text example is above if

File: source/text/parser.go (hunk 5 above)
Comment (sexyhamster) [reply source/text/parser.go]: fixed

File: source/text/parser_test.go (hunk 2 above)
Comment (sexyhamster) [reply source/text/parser_test.go]: my bad, was debugging, removed

**Pull request merged**